simulator/infra/ports/module.h

```diff
@@ -9,9 +9,13 @@
  
 #include <infra/log.h>
 #include <infra/ports/ports.h>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/json_parser.hpp>
 
 #include <unordered_set>
 
+namespace pt = boost::property_tree;
```

Comment:
Same for the alias. Better to move it the inner scope of *.cpp file, since it is very likely to have a name conflict.